plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -764,6 +774,14 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                 principalPrivileges);
     }
 
+    private static void checkChangeDataFeedColumnNames(Set<String> columnNames)
```

Comment:
The method is always invoked when caller has a list of ColumnMetadata
pass it here, so the call sites are simpler to read (removing need for `.stream().map(ColumnMetadata::getName).collect(Collectors.toSet())`)